test/common/http/http2/conn_pool_test.cc

```diff
@@ -114,6 +114,20 @@ class Http2ConnPoolImplTest : public TestBase {
   // Asserts that the provided requests receives onPoolFailure.
   void expectStreamReset(ActiveTestRequest& r);
 
+  /** Closes a test client
```

Comment:
nit: formatting, also please newline between method and the next comment, same below.